### PR TITLE
header updated

### PR DESCRIPTION
**Pull request by Muhammad-Ali-Ahmad**

### CODE REVIEW

File: resources/govuk-template.mustache.html [hunk 1]
```diff
@@ -158,19 +158,47 @@
...
-                    <div class="header-logo">
-                        <a href="https://www.gov.uk/" title="Go to the GOV.UK homepage" data-journey-click="header:Click:GOV UK" id="logo" class="content">
-                            <img src="/template/assets/images/gov.uk_logotype_crown.png" alt=""> GOV.UK
-                        </a>
-                    </div>
+                {{#newGovukLogo}}
+                <div class="govuk-header__logo">
+                    <a href="https://www.gov.uk" class="govuk-header__link govuk-header__link--homepage">
+                        <span class="govuk-header__logotype">
+                          <!--[if gt IE 8]><!-->
```
Comment (Luka-Abey) [inline resources/govuk-template.mustache.html:165]: do we need this/does hmrc still support IE8?

Comment (Luka-Abey) [reply resources/govuk-template.mustache.html:165]: https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices

**Review by Christian-QA — APPROVED** [review]

**Review by russellthorn — APPROVED** [review]

File: resources/govuk-template.mustache.html (hunk 1 above)
Comment (Muhammad-Ali-Ahmad) [reply resources/govuk-template.mustache.html:165]: Yes you are right @Luka-Abey but I coped it as it was in design system https://design-system.service.gov.uk/components/header/ :)

Comment (Luka-Abey) [reply resources/govuk-template.mustache.html:165]: fantastic, thanks

**Review by Luka-Abey — APPROVED** [review]

**Pull request merged**